python/cudf/cudf/core/series.py

```diff
@@ -1278,15 +1278,12 @@ def to_array(self, fillna=None):
     def isnull(self):
         """Identify missing values in a Series.
         """
-        if not self.has_null_mask:
-            return Series(
-                cudautils.zeros(len(self), np.bool_),
-                name=self.name,
-                index=self.index,
-            )
-
-        mask = cudautils.isnull_mask(self.data, self.nullmask.mem)
-        return Series(mask, name=self.name, index=self.index)
+        result = Series(
+            libcudf.unaryops.isNULL(self._column),
+            name=self.name,
+            index=self.index,
+        )
```

Comment:
Can we create the `isnull`, `isna`, `notnull`, and `notna` functions as Column functions and then call those functions from the Series / Index classes?